lib/index.js

```diff
@@ -20,6 +21,12 @@ var numberFormatPHPJS = require( './number-format' );
 var decimal_point_translation_key = 'number_format_decimals',
 	thousands_sep_translation_key = 'number_format_thousands_sep';
 
+var hashers = [
+	function( options ) {
+		return options;
+	}
```

Comment:
What does the function at `0` do? Does it deserve a comment or should it be obvious?

Reply:
Thanks, I added a comment.